common/changes/sparo/chore-update-sparo-profile-template_2024-03-14-23-58.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "sparo",
+      "comment": "Add more comments in sparo profile template",
```

```suggestion
      "comment": "Add more comments in Sparo profile template",
```

We should always capitalize Sparo unless we're specifically referring to the shell binary `sparo`.